Changelog.md

```diff
@@ -6,6 +6,9 @@ Language Features:
 Compiler Features:
  * NatSpec: Add fields "kind" and "version" to the JSON output.
  * Commandline Interface: Prevent some incompatible commandline options from being used together.
+ * Commandline Interface: Adds new option ``--hyperlinks`` and ``--no-hyperlinks`` for improved
+   diagnostics formatting. File names will be conditionally formatted as hyperlink in supporting
+   terminals.
```

Comment:
I think we don't do linebreaks in changelog

Reply:
Actually we do. At least I was basing this message on the one with regards to `--color` / `--no-color`.

Reply:
Please don't break lines in the changelog. This entry could also be shortened a little.

Reply:
ok